### ISSUE TITLE
Contributing question 

### ISSUE THREAD

**Issue by Qd0t** [emipa606/GrowingZonePriorities#1]
Hi,

I made some of the UI aspects neater and wanted to contribute but I don't know what I'm doing on GitHub lol, I created a fork but I don't know what to do with it. Do you know if I can attach the files I modified and some custom textures I've made?

**Comment by emipa606** [comment]
You can either do a Pull Request, where you update your fork with the changed files.
Or you can just attach the updated files to this issue.

**Comment by Qd0t** [comment]
[Neater UI - files changed.zip](https://github.com/emipa606/GrowingZonePriorities/files/10060528/Neater.UI.-.files.changed.zip)

I have attached the files (had to zip them) you should be able to just paste all this into mod directory for changes to apply. Not sure about the fork stuff, will look into that later lol.

I included the source files in there too, both cs and psd for textures. I changed the icon of buttons and put a condition for the action function (where it creates the window), also added texture library in the main cs file.

**Comment by emipa606** [comment]
There seem to be an issue with this.
If you select multiple zones, open the change prio-window and then cancel it, the gizmos will be broken.

**Comment by Qd0t** [comment]
There is a similar bug in the vanilla, I tried to force the Normal priority for slider menu, just forgot to test it lol. The gizmos prob failed to render since I'm using current value to fetch the icon. Anyhow I redesigned the multi selection gizmo with float menu. It's not ideal since it commits on button click but I don't want to mess with the checks that set the current value to negative integers since I'm not really sure why they are there to begin with. 
[Neater UI - files changed.zip](https://github.com/emipa606/GrowingZonePriorities/files/10092315/Neater.UI.-.files.changed.zip)
